nearcore/src/lib.rs

```diff
@@ -570,7 +572,12 @@ pub fn recompress_storage(home_dir: &Path, opts: RecompressOpts) -> anyhow::Resu
     info!(target: "recompress", src = %src_dir.display(), dest = %opts.dest_dir.display(), "Recompressing database");
     let src_store = create_store_with_config(
         &src_dir,
-        StoreConfig { read_only: true, enable_statistics: false },
+        StoreConfig {
+            read_only: true,
```

Comment:
we should have a constructor instead (where you just pass config.store as parameter)